activiti-cloud-services-runtime-bundle/activiti-cloud-services-identity-basic/src/test/java/org/activiti/cloud/services/identity/basic/BasicUserGroupManagerIT.java

```diff
@@ -0,0 +1,31 @@
+//package org.activiti.cloud.services.identity.basic;
+//
+//import org.activiti.runtime.api.identity.UserGroupManager;
+//import org.junit.Test;
+//import org.junit.runner.RunWith;
+//import org.springframework.beans.factory.annotation.Autowired;
+//import org.springframework.boot.test.context.SpringBootTest;
+//import org.springframework.context.annotation.ComponentScan;
+//import org.springframework.test.context.junit4.SpringRunner;
+//
+//import static org.assertj.core.api.Assertions.assertThat;
+//
+//@RunWith(SpringRunner.class)
+//@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+//public class BasicUserGroupManagerIT {
```

Comment:
@Salaboy what is the status of this test?

Reply:
@erdemedeiros sorry.. please remove.. I've created it but then I realised that we don't need it.

Reply:
Moved to Activiti/activiti-spring-identity